docs/modules/ROOT/pages/producing.adoc

```diff
@@ -25,9 +25,10 @@ state, and your generator function now returns a new state on each round.
 For instance, you could use an `int` as the state:
 
 .Example of state-based `generate`
```

Comment:
The `generate` does not render correctly as code.

Reply:
I cannot find a solution for the moment.
what I see is that when also generating the doc into PDF, then the `generate` part is rendered:

![pdf](https://github.com/reactor/reactor-core/assets/650758/b757da22-1b40-4333-802a-3f4b310f9adc)

Now, I have tried to change the backquotes with "#", like this:

```
.Example of state-based #generate#
```

and `generate` is then rendered like this:

![Screenshot 2024-05-15 at 19 20 31](https://github.com/reactor/reactor-core/assets/650758/fa6b905d-3272-40fa-9476-66358cffb019)

are you ok to go with this ? else, I think we need to address this issue later, using another PR ?

Reply:
I don't think the green background works. Let's see if this can be improved afterwards.

Reply:
Have created #3813 issue to not forget about this.